v2/models/propertyvalue.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+const (
+	// Base64Encoding : the float value is represented in Base64 encoding
+	Base64Encoding = "Base64"
+	// ENotation : the float value is represented in eNotation
+	ENotation = "eNotation"
```

Comment:
These are repeated in the DTOs. Should only need them in on place, not both.

Reply:
Remove from DTO.